homeassistant/components/vicare/types.py

```diff
@@ -64,6 +64,55 @@ def from_ha_preset(
 }
 
 
+class VentilationMode(enum.StrEnum):
+    """ViCare ventilation modes."""
+
+    PERMANENT = "permanent"  # on, speed controlled by program (levelOne-levelFour)
+    VENTILATION = "ventilation"  # activated by schedule
+    SENSOR_DRIVEN = "sensorDriven"  # activated by schedule, override by sensor
+    SENSOR_OVERRIDE = "sensorOverride"  # activated by sensor
```

Comment:
Change string to the translation keys 

`PERMANENT = "permanent"  # on, speed controlled by program (levelOne-levelFour)
VENTILATION = "ventilation"  # activated by schedule
SENSOR_DRIVEN = "sensor_driven"  # activated by schedule, override by sensor
SENSOR_OVERRIDE = "sensor_override"  # activated by sensor`